doc/source/whatsnew/v1.0.0.rst

```diff
@@ -45,7 +45,7 @@ Backwards incompatible API changes
 Other API changes
 ^^^^^^^^^^^^^^^^^
 
--
+- :meth:`pandas.Series.str.cat` now defaults to aligning ``others``, using ``join='left'`` (:issue:`27611`)
```

Comment:
how is this an api change? isn't it just a deprecation removal?

Reply:
I thought that changing the default of a kwarg qualifies more as an API change than a removal, but I don't mind either way. Let me know if you want me to shift it.

Reply:
We have a "Removal of prior version deprecations/changes" section, which IIUC is appropriate here.